angular/projects/spark-angular/schematics/ng-add/index.ts

```diff
@@ -0,0 +1,179 @@
+import {
+  addModuleImportToRootModule,
+  getAppModulePath,
+  getProjectFromWorkspace,
+  getProjectIndexFiles,
+  getProjectMainFile,
+  getProjectStyleFile,
+  hasNgModuleImport,
+} from '@angular/cdk/schematics';
+import { chain, Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
+import { getWorkspace } from '@schematics/angular/utility/config';
+import {
+  addPackageJsonDependency,
+  NodeDependency,
+  NodeDependencyType,
+  removePackageJsonDependency,
+} from '@schematics/angular/utility/dependencies';
+
+import { Schema } from './schema';
+import { addHtmlClass } from './html-manipulation';
+
+/**
+ * Constants
+ */
+const sparkAngularModuleName = 'SparkAngularModule';
+const sparkAngularModuleImport = '@sparkdesignsystem/spark-angular';
+const sparkAngularJavaScriptClass = 'sprk-u-JavaScript';
+const browserAnimationsModuleName = 'BrowserAnimationsModule';
+const browserAnimationsModuleImport = '@angular/platform-browser/animations';
+const noopAnimationsModuleName = 'NoopAnimationsModule';
+
+/**
+ * Scaffolds the basics of a Spark Angular application, this includes:
+ *  - Adds the spark and spark-angular dependencies to package.json
+ *  - Adds the SparkAngularModule to the root module of the specified project
+ *  - Adds the BrowserAnimationsModule to the root module of the specified project
+ *  - Adds the Spark stylesheets to the project style file
+ *  - Adds the Spark class to the project html tag
+ */
+export function ngAdd(options: Schema): Rule {
+  return chain([
+    addSparkDependencies(options),
+    addSparkModule(options),
+    addAnimationsModule(options),
+    addSparkStyles(options),
+    addSparkClass(options),
+  ]);
+}
+
+/**
+ * Adds the spark and spark-angular dependencies to package.json
+ */
+function addSparkDependencies(_options: Schema): Rule {
+  return (tree: Tree, context: SchematicContext) => {
+    // Schematics automatically adds to "dependencies" but it should be "devDependencies"
+    removePackageJsonDependency(tree, '@sparkdesignsystem/spark-angular');
+
+    const dependencies: NodeDependency[] = [
+      { name: '@sparkdesignsystem/spark', overwrite: true, type: NodeDependencyType.Dev, version: '^13.0.0' },
```

Comment:
```suggestion
      { name: '@sparkdesignsystem/spark', overwrite: true, type: NodeDependencyType.Dev, version: '^13.1.0' },
```
We had a release yesterday.